atom/browser/net/url_request_async_asar_job.cc

```diff
@@ -71,14 +71,23 @@ void URLRequestAsyncAsarJob::StartAsync(std::unique_ptr<base::Value> options,
   }
 
   std::string file_path;
+  response_headers_ = new net::HttpResponseHeaders("HTTP/1.1 200 OK");
   if (options->is_dict()) {
-    auto* path_value =
-        options->FindKeyOfType("path", base::Value::Type::STRING);
-    if (path_value)
-      file_path = path_value->GetString();
+    base::DictionaryValue* headersValue = nullptr;
+    base::DictionaryValue* dict =
+        static_cast<base::DictionaryValue*>(options.get());
+    dict->GetString("path", &file_path);
```

Comment:
This introduces a minor regression of going back to deprecated API:

```c++
// DEPRECATED, use Value::FindPath(path) and Value::GetString() instead.
bool GetString(StringPiece path, std::string* out_value) const;
```

Could you keep the previous `FindKeyOfType` implementation?

Reply:
Ah, hadn't noticed that was deprecated. Fixed!